tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/jvm/ForeachArrayTests.java

```diff
@@ -223,6 +228,26 @@ public void testEmptyArray() {
         BRunUtil.invoke(program, "testEmptyArray");
     }
 
+    @Test
+    public void testLoopEmptyTuple() {
```

Comment:
Also, is this the correct file for these tests? It says ForeachArrayTests, may be the tuple tests are somewhere else?

Reply:
Updated 376dc2f1f9bf11df4cd92cf83d6edff3cba48765